nservicebus/extending/queuecreation.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Declaring and creating queues
+summary: Explains how to declare additional queues and create them
+tags:
+ - Queue
+ - QueueBindings
+ - Queue creation
+---
+
+NServiceBus queue creation process is pluggable. The process consists of two phases:
+
+* Declaring queues which need to be created
+* Creating the declared queues.
+
+This page describes the declaration and creation process of queues. Usually only transports need to deal with queue declaration and creation.
+
+## Declaration
+
+Queues should be declared during the Setup phase of a Feature. Read more how to write a Feature on the [Feature documentation page](/nservicebus/pipeline/feature.md).
```

Comment:
drop the "Read more how to write a Feature on the" and just make the previous "Feature" text a link